linker/Mono.Linker.Steps/OutputStep.cs

```diff
@@ -55,6 +79,29 @@ protected override void ProcessAssembly (AssemblyDefinition assembly)
 			OutputAssembly (assembly);
 		}
 
+		private bool isR2R(ModuleDefinition module) {
```

Comment:
Please use existing coding style.
- no private
- Useful name IsRreadyToRun
- Space between name and parens
- Redundant parens in return statement
- { at new line
- method should be static